plugins/reporters/spdx/src/funTest/kotlin/SpdxDocumentReporterFunTest.kt

```diff
@@ -273,13 +273,16 @@ private val ortResult = OrtResult(
     scanner = scannerRunOf(
         Identifier("Maven:first-package-group:first-package:0.0.1") to listOf(
             ScanResult(
-                provenance = ArtifactProvenance(
-                    sourceArtifact = RemoteArtifact(
-                        url = "https://some-host/first-package-sources.jar",
-                        hash = Hash.create("0000000000000000000000000000000000000000")
-                    )
+                provenance = RepositoryProvenance(
+                    vcsInfo = VcsInfo(
+                        type = VcsType.GIT,
+                        revision = "master",
+                        url = "ssh://git@github.com/path/first-package-repo.git",
```

Comment:
You changed the VCS URL without updating line 50 in[plugins/reporters/spdx/src/funTest/assets/spdx-document-reporter-expected-output.spdx.yml and line 40 in plugins/reporters/spdx/src/funTest/assets/spdx-document-reporter-expected-output.spdx.json.

Reply:
In SPDX we produce 3 packages per ID:

1. binary package
2. source artifact package
3. VCS package

The lines you mention refer to 1. which uses the binary artifact URL. So, a change in these lines would be unrelated / incorrect. (I've just verified again the test is green)